addons/coupon/security/ir.model.access.csv

```diff
@@ -0,0 +1,9 @@
+id,name,model_id/id,group_id/id,perm_read,perm_write,perm_create,perm_unlink
+access_program_salesman,Coupon Program (Salesperson),model_coupon_program,sales_team.group_sale_salesman,1,0,0,0
+access_program_manager,Coupon Program (Manager),model_coupon_program,sales_team.group_sale_manager,1,1,1,1
+access_applicability_salesman,Coupon Program (Salesperson),model_coupon_rule,sales_team.group_sale_salesman,1,0,0,0
```

Comment:
please adapt my name suggestion to the underlying model :p

so `Coupon Rule (Salesperson)`, `Coupon (Salesperson)`, `Coupon Reward (Salesperson)`, etc. Basically `Readable Model Name (Access Rights Level)`.

Reply:
The rest of the changes looks good :+1:

Reply:
done